extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -122,7 +122,6 @@ public function getVideos() {
 	protected function getCacheKey() {
 		$cacheKey = wfSharedMemcKey(
```

Comment:
Sure, and we can change them as well so that ultimately, all overrides only append their specific parameters to the key, but the parent makes the single call to `wfMemcKey`; @owend made a comment on this too few days ago saying this is the more common way of making cache keys.